docs/PROCESS

```diff
@@ -0,0 +1,36 @@
+# Process
```

Comment:
I guess it's not really about processes?

Reply:
Would it make more sense to add all of this under "How to contribute" on [CONTRIBUTING](https://github.com/spruceid/spruceid.dev/blob/main/contributing)?
Or is there a better word than "Process"?

Reply:
+1 to put it in this repo's CONTRIBUTING.md